flink-doris-connector/src/main/java/org/apache/doris/flink/sink/schema/SQLParserSchemaManager.java

```diff
@@ -267,11 +280,28 @@ private String processRenameColumnOperation(
     }
 
     @VisibleForTesting
-    public String extractDefaultValue(List<String> columnSpecs) {
+    public String extractDefaultValue(String dateType, List<String> columnSpecs) {
         if (CollectionUtils.isEmpty(columnSpecs)) {
             return null;
         }
-        return extractAdjacentString(columnSpecs, DEFAULT);
+        String adjacentDefaultValue = extractAdjacentString(columnSpecs, DEFAULT);
+        return parseDorisDefaultValue(dateType, adjacentDefaultValue);
+    }
+
+    private String parseDorisDefaultValue(String dateType, String defaultValue) {
+        if (Objects.isNull(defaultValue)) {
+            return null;
+        }
+        // In doris, DATETIME supports specifying the current time by default through
+        // CURRENT_TIMESTAMP.
+        if (dateType.startsWith(DorisType.DATETIME) || dateType.startsWith(DorisType.DATETIME_V2)) {
+            for (String timeValue : sourceConnectorTimeValues) {
+                if (timeValue.equalsIgnoreCase(defaultValue)) {
+                    return DORIS_CURRENT_TIMESTAMP;
+                }
+            }
```

Comment:
Why not   use sourceConnectorTimeValues.contains?

Reply:
There are cases of upper and lower case